pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go

```diff
@@ -1151,3 +1323,14 @@ func setCgroupVersionDuringTest(version CgroupVersion) {
 		return version == cgroupV2
 	}
 }
+
+func setSwapControllerAvailableDuringTest(available bool) func() {
+	original := swapControllerAvailable
+	swapControllerAvailable = func() bool {
+		return available
+	}
+
+	return func() {
+		swapControllerAvailable = original
+	}
```

Comment:
Isn't doing the following enough?
```go
swapControllerAvailable = func() bool {
	return available
}
```

I'm pretty sure that all test cases run independently, so no need to revert to the original function value

Reply:
Yes, it may be enough. I was thinking that there may be some issue in the case of failures, for example. If you feel strongly about this, I can revert.

Reply:
I don't feel strongly about it, but if code can be simplified for "free" it's always a win :)
Feel free to leave it as is, we can also circle back to it in future follow-ups.

Thank you!

Reply:
OK. Leaving as is for now. My motivation for doing this was to align with how the feature gates are handled in test.